p2p/simulations/http.go

```diff
@@ -423,6 +454,25 @@ func (s *Server) StreamNetworkEvents(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+func (s *Server) evalMsgFilter(activeMsgFilters map[string][]uint64, event *Event) bool {
+	matches := false
+	//client needs to at least pass "filterProtocol" to enable filtering
+	//if not, no messages will be delivered at all
+	if len(activeMsgFilters) > 0 {
```

Comment:
indeed, but I thought it's slightly faster to have this check first instead of having `go` searching the complete map first to see if there is any match at all when there are no filters enabled anyway. I vote to keep this.